fix: ticket section render issue using slider component

toffee-k21 opened the pull request:
<!--   Thank you for your contribution. Before you submit the pull request:
1. Follow our contribution guidelines
2. Test your changes and attach their results to the pull request.
3. Update the relevant documentation.
-->

**Description**

fixes #395 
fixed card rendering issue in the Ticket section

ashmit-coder reviewed (changes requested):
![image](https://github.com/user-attachments/assets/d69929fa-3eae-49c9-99bb-6992daa163f9)
@toffee-k21 the tickets are breaking in the on compressed view.

ashmit-coder reviewed (changes requested):
Also add a test in the cypress test suite, for the changes you have made.

toffee-k21 commented:
> Also add a test in the cypress test suite, for the changes you have made.

Hey @ashmit-coder, I have made the changes you suggested and also added Cypress tests for the changes I made.

Additionally, there was an issue with the overflow of the slider component, which I have fixed. Below is the video regarding that

https://github.com/user-attachments/assets/7907fd28-919f-494e-ab79-ae3cccbc1e47

https://github.com/user-attachments/assets/3f18713c-171f-44da-8081-672d19e45bbb

AceTheCreator reviewed (changes requested):
good work @toffee-k21, few suggested changes

- I noticed the venues on the landing is aligned to the left on desktop view
- I think you should remove the description in the ticket card and just keep it simple

toffee-k21 commented:
Thanks @AceTheCreator for your feedback, I have implemented what you suggested. Let me know if there's anything else you'd like me to adjust!

ashmit-coder commented:
> Thanks @AceTheCreator for your feedback, I have implemented what you suggested. Let me know if there's anything else you'd like me to adjust!

cc: @AceTheCreator

AceTheCreator commented:
@toffee-k21, sorry for the late response. Can you resolve the conflict in this PR

toffee-k21 commented:
Hey @AceTheCreator, I had to create a new PR #547 due to sync issues. Let me know if everything looks good!
